shared/whats-new/index.stories.tsx

```diff
@@ -0,0 +1,407 @@
+import * as React from 'react'
+import * as Sb from '../stories/storybook'
+import * as Kb from '../common-adapters'
+import * as Styles from '../styles'
+import {Version, VersionTitle, VersionProps} from './versions'
+import {currentVersion, lastVersion, lastLastVersion} from '../constants/whats-new'
+import HeaderIcon from './icon/'
+import NewFeatureRow from './new-feature-row'
+import WhatsNew from '.'
+
+const darkModeImage = require('../images/releases/storybook/dark.png')
```

Comment:
typo, needs to be `dark-mode.png`

Reply:
@cjb actually I found that this line was matching the file path and turning it into an `esModule`

https://github.com/keybase/client/blob/master/shared/storybook/webpack.config.js#L35-L38

Reply:
It shows up as a broken image in storybook for me

Reply:
Good to know.